Tools/CISupport/ews-build/steps.py

```diff
@@ -3073,15 +3073,6 @@ def getResultSummary(self):
         return {'step': message}
 
 
-class RunWebKitPyPython2Tests(WebKitPyTest):
-    language = 'python2'
-    name = 'webkitpy-tests-{}'.format(language)
-    description = ['webkitpy-tests running ({})'.format(language)]
-    jsonFileName = 'webkitpy_test_{}_results.json'.format(language)
-    logfiles = {'json': jsonFileName}
-    command = ['python', 'Tools/Scripts/test-webkitpy', '--verbose', '--json-output={0}'.format(jsonFileName)]
-
-
 class RunWebKitPyPython3Tests(WebKitPyTest):
     language = 'python3'
     name = 'webkitpy-tests-{}'.format(language)
```

Comment:
now that there is no more python2 in this code, we can consider removing explicit python3 from these step names.

Reply:
Good point, let me do that...